source/common/router/router.cc

```diff
@@ -416,13 +416,9 @@ void Filter::maybeDoShadowing() {
 void Filter::onRequestComplete() {
   downstream_end_stream_ = true;
   downstream_request_complete_time_ = std::chrono::steady_clock::now();
```

Comment:
Can this be computed from one of the existing time values on the request info?

Reply:
Totally, though if we convert the return values to durations, we won't have access to the time points.

Reply:
Oh I take that back, we immediately calculate a duration from it. LastDownstreamTxByteSent should be good.

Reply:
Leaving these in for now. They're a bit tangled in the tests and will benefit from being done in a separate PR, if that's fine?

Reply:
Yes, I think that's fine